src/main/java/appeng/client/gui/implementations/GuiMEMonitorable.java

```diff
@@ -139,7 +140,7 @@ else if( te instanceof IMEChest )
 		{
 			this.myName = GuiText.Chest;
 		}
-		else if( te instanceof PartTerminal )
+		else if( te instanceof AbstractPartTerminal )
```

Comment:
Probably another task related to rework the GUI as this will affect mostly unrelated classes (compared to panels)